llap-tez/src/java/org/apache/hadoop/hive/llap/tezplugins/LlapTaskSchedulerService.java

```diff
@@ -2500,11 +2504,11 @@ public void shutdown() {
 
     private int numPreemptedTasks = 0;
```

Comment:
Is numPreemtedTasks used/read anywhere?

Reply:
Hmm, I don't see any read on it. Let me wipe it out.